tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "moduleResolution": "node",
     "forceConsistentCasingInFileNames": true,
     "outDir": "dist",
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "noEmit": true
```

Comment:
What does this do? Wondering if this can be put into an ignored config for you if it's a preference thing.

Reply:
This just makes it so running `tsc` doesn't product `.js`. Useful to avoid the `.js + .ts` issue i ran into for verification from before.

Reply:
But let me remove this for now just to keep things simple.